### PR TITLE
Allow import typing

### PR DESCRIPTION
**Pull request by jayvdb**
Previously only `from typing import ..` was supported.

### CODE REVIEW

File: py2many/clike.py [hunk 1]
```diff
@@ -208,6 +208,11 @@ def _typename_from_type_node(self, node) -> Union[List, str, None]:
             return get_id(node)
         elif isinstance(node, ast.Tuple):
             return [self._typename_from_type_node(e) for e in node.elts]
+        elif isinstance(node, ast.Attribute):
+            node_id = get_id(node)
+            if node_id.startswith("typing."):
```
Comment (adsharma) [inline py2many/clike.py:213]: One could also write:

```
from typing import List as typ_list
```
A more general solution would involve some type of eval/__import__ of the statement above and then looking up typing.List in a dict, like what we're doing in class_for_typename/_func_for_lookup.

Could use remove_prefix post 3.9:

https://www.python.org/dev/peps/pep-0616/

File: tests/cases/binit.py [hunk 2]
```diff
@@ -2,10 +2,10 @@
 
 # http://rosettacode.org/wiki/Bin_given_limits
 
-from typing import List
```
Comment (adsharma) [inline tests/cases/binit.py:5]: For the case where only one thing is imported, the old code was more concise.
Why not add a new test and leave these cases alone?

Comment (jayvdb) [reply tests/cases/binit.py]: It isnt more consise
```
from typing import List;List
import typing;typing.List
```
Not sure why you dont want to use the existing test cases.  A new cases just for this import is hardly worth it, and ive run out of time I can allocate to this project this weekend.

File: py2many/clike.py (hunk 1 above)
Comment (jayvdb) [reply py2many/clike.py:213]: sure.  there are lots of problems, and this only solves one of them, and does so without the overhead of eval/import.  Created https://github.com/adsharma/py2many/issues/409 for a fallback to more difficult cases.

**Pull request merged**